gradle-plugin/it/src/test/java/org/eclipse/jkube/gradle/plugin/tests/ServiceAccountNameViaGroovyDSLIT.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.gradle.plugin.tests;
+
+import net.minidev.json.parser.ParseException;
+import org.eclipse.jkube.kit.common.ResourceVerify;
+import org.gradle.testkit.runner.BuildResult;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+class ServiceAccountNameViaGroovyDSLIT {
+  @RegisterExtension
+  private final ITGradleRunnerExtension gradleRunner = new ITGradleRunnerExtension();
+
+  static Stream<Arguments> testInput() {
+    return Stream.of(
+//        arguments("no-fragment", new String[] {}),
+//        arguments("fragment", new String[] {"-Pjkube.resourceDir=./regularFragmentDir"}),
```

Comment:
From the test expectations, the assumption here is that a fragment with an unrelated field has no action over the serviceAccountName field, right?

Could we state this more clearly? A simple "fragment" seems to convey the opposite assumption (I have a fragment with a service account -> what we test in the next case). So maybe an `unrelated-fragment` name might be more suitable.

Reply:
I've renamed`fragment` -> `unrelated-fragment`